packages/web/src/utils/validators.ts

```diff
@@ -0,0 +1,184 @@
+import { BigNumber } from 'bignumber.js'
+import { ValidatorsListProps } from 'src/dev/ValidatorsList'
+import { styles } from 'src/dev/ValidatorsListStyles'
+import { weiToDecimal } from 'src/utils/utils'
+
+export const localStoragePinnedKey = 'pinnedValidators'
+
+export interface Edges<T> {
+  edges: Array<{
+    node: T
+  }>
+}
+
+export interface CeloValidatorGroup {
+  account: {
+    address: string
+    lockedGold: string
+    name: string
+    usd: string
+    claims: Edges<{
+      verified: boolean
+      element: string
+    }>
+  }
+  accumulatedActive: string
+  accumulatedRewards: string
+  affiliates: Edges<{
+    account: {
+      claims: Edges<{
+        verified: boolean
+        element: string
+      }>
+    }
+    address: string
+    attestationsFulfilled: number
+    attestationsRequested: number
+    lastElected: number
+    neverElected: boolean
+    lastOnline: number
+    lockedGold: string
+    name: string
+    score: string
+    usd: string
+  }>
+  commission: string
+  numMembers: number
+  receivableVotes: string
+  rewardsRatio: string
+  votes: string
+}
+
+export interface CeloGroup {
+  id: number
+  elected: number
+  online: number
+  total: number
+  uptime: number
+  attestation: number
+  name: string
+  address: string
+  usd: number
+  celo: number
+  receivableRaw: number
+  receivableVotes: string
+  votesRaw: number
+  votes: string
+  votesAbsolute: string
+  commission: number
+  rewards: number
+  rewardsStyle: any
+  numMembers: number
+  claims: string[]
+  validators: Array<{
+    name: string
+    address: string
+    usd: number
+    celo: number
+    elected: boolean
+    neverElected: boolean
+    online: boolean
+    uptime: number
+    attestation: number
+    claims: string[]
+  }>
+}
+
+export function isPinned({ address }: any) {
```

Comment:
any chance this can be typed more explicitly?

Reply:
Yes!  Added address type, let me know if that looks sufficient